dubbo-remoting/dubbo-remoting-http/src/main/java/org/apache/dubbo/remoting/http/HttpHandler.java

```diff
@@ -34,6 +32,6 @@ public interface HttpHandler {
      * @throws IOException
      * @throws ServletException
      */
-    void handle(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException;
+    void handle(REQUEST request, RESPONSE response) throws IOException, ServletException;
```

Comment:
will throw an `java.lang.NoClassDefFoundError` if the classpath does not have `ServletException`

Reply:
ok,i will fix